<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>m2Reconstruction3DControls</class>
 <widget class="QWidget" name="m2Reconstruction3DControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>1002</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Image selection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnUpdateList">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Update list</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dataSelection">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSwitch">
         <property name="text">
          <string>Switch column</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Remove selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxCoRegistrationToSelected">
     <property name="text">
      <string>Coregister all images to the selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxOmitDeformable">
     <property name="text">
      <string>Rigid only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxUseFixedMask">
     <property name="text">
      <string>Use fixed mask</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxUseMovingMask">
     <property name="text">
      <string>Use moving mask</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxUseMovingMaskDeform">
     <property name="text">
      <string>Use moving mask (deformable)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxUseFixedMaskDeform">
     <property name="text">
      <string>Use fixed mask (deformable)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnEditParameterFiles">
     <property name="text">
      <string>Edit elastix parameter files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>z-spacing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxZSpacing">
       <property name="suffix">
        <string> Î¼m</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rigid max iterations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="RigidMaxIters">
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Deformable max iterations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="DeformableMaxIters">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnStartStacking">
     <property name="text">
      <string>Start stacking</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Export all centroids as 3D NRRD files:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="centroidListSelector" native="true"/>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="stackImageSelector" native="true"/>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnStartStackExport">
     <property name="text">
      <string>Start stack export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
